src/aggregators/bracketing.jl

```diff
@@ -14,15 +14,18 @@ end
 BracketData{T1, T2}() where {T1, T2} = BracketData(T1(0.1), T2(25), T2(4))
 
 # support either vectors of data for each field, or scalars
+"Get fluctuation rate of species i."
```

Comment:
Unless you are writing these to be comments that are visible to users via the REPL, in which case they should document using the standard documentation format that also describes what the arguments are and the function signature, I'd suggest using a hash.

Reply:
Any strings you use become visible to users who type "? func_name", whereas I don't think hash-based comments do (so we can think of them as internal only documentation).

Reply:
Switched to use `#`.